src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/FootnoteServiceTests.cs

```diff
@@ -538,11 +544,490 @@ void AssertFootnoteDetailsCopiedCorrectly(Footnote originalFootnote, Footnote ne
             }
         }
 
+        [Fact]
+        public async Task DeleteFootnote()
+        {
+            var release = new Release();
+
+            var subject = new Subject
+            {
+                Id = Guid.NewGuid(),
```

Comment:
Do we need to generate ids for the Subject/Filter/Indicator models?

Think we could remove all of these ids in favour of linking the models directly (would save a bunch of lines). Seems like we have done this for `indicatorGroup`, but not the others?

Would do this for the other test cases further down as well.